### PR TITLE
[INLONG-8114][Manager] Fix the NEP that may appear when building Properties.

### LINKED ISSUE #8114: [Bug][Manager] Fix the NEP that may appear when building Properties.
### What happened

There is a null value in the Properties map, but the direct toString operation leads to NEP

### What you expected to happen

There is a null value in the Properties map, but the direct toString operation leads to NEP.

### How to reproduce

There is a null value in the Properties map, but the direct toString operation leads to NEP.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chestnut-c**
- Fixes #8114

### Motivation

Fix the NEP that may appear when building Properties , Filter empty values ​​to prevent NEP, because empty values ​​are meaningless to pass downstream.

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 package org.apache.inlong.manager.pojo.sort.util;
 
+import java.util.Objects;
 import org.apache.inlong.common.enums.DataTypeEnum;
 import org.apache.inlong.manager.common.consts.SinkType;
 import org.apache.inlong.manager.common.enums.ErrorCodeEnum;
```
Comment (gong) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java:23]: you can rebase master and exec `mvn spotless:apply` for code style

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java [hunk 2]
```diff
@@ -563,6 +564,7 @@ private static List<FieldInfo> parseFieldInfos(List<StreamField> streamFields, S
      */
     private static Map<String, String> parseProperties(Map<String, Object> properties) {
         return properties.entrySet().stream()
+                .filter(v -> Objects.nonNull(v.getValue()))
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java:567]: Suggest use `StringUtils.isNotBank()`.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java [hunk 3]
```diff
@@ -117,6 +118,7 @@ public static LoadNode createLoadNode(StreamSink streamSink, Map<String, StreamF
                 .collect(Collectors.toList());
         List<FieldRelation> fieldRelations = parseSinkFields(streamSink.getSinkFieldList(), constantFieldMap);
         Map<String, String> properties = streamSink.getProperties().entrySet().stream()
+                .filter(v -> Objects.nonNull(v.getValue()))
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java:121]: Suggest use `StringUtils.isNotBank()`.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java (hunk 1 above)
Comment (chestnut-c) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java]: > You can rebase master and exec `mvn spotless:apply` for code style

ok  thanks

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java (hunk 2 above)
Comment (chestnut-c) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java:567]: > Suggest use `StringUtils.isNotBank()`.

Value is of type Object，so i use Objects.nonNull(Object obj)

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java (hunk 3 above)
Comment (chestnut-c) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java:121]: > Suggest use `StringUtils.isNotBank()`.

Value is of type Object，so i use Objects.nonNull(Object obj)

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]
LGTM

**Pull request merged**